src/lib/json-rpc.ts

```diff
@@ -1,3 +1,5 @@
+import { v4 as uuidv4 } from 'uuid';
+
 export type JsonRpcId = string | number | null;
 export type JsonRpcParams = any;
```

Comment:
What do you think if we just remove this line, the layer of abstraction seems to be useless?

Reply:
Good catch! Id was optional at one point but realized I wanted to make it a required param, so yeah this is useless now.

Reply:
Done

Reply:
Sanity: maybe miscommunication: I was referring to removal of `export type JsonRpcParams = any;`